sql/api/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -8119,6 +8119,41 @@ object functions {
       secs: Column): Column =
     Column.fn("make_timestamp", years, months, days, hours, mins, secs)
 
+  /**
+   * Try to create a timestamp from years, months, days, hours, mins, secs and timezone fields.
+   * The result data type is consistent with the value of configuration `spark.sql.timestampType`.
+   * The function returns NULL on invalid inputs.
+   *
+   * @group datetime_funcs
+   * @since 4.0.0
+   */
+  def try_make_timestamp(
+      years: Column,
+      months: Column,
+      days: Column,
+      hours: Column,
+      mins: Column,
+      secs: Column,
+      timezone: Column): Column =
+    Column.fn("try_make_timestamp", years, months, days, hours, mins, secs, timezone)
+
+  /**
+   * ANSI-compliant version of the make_timestamp function. Try to create a timestamp from years,
```

Comment:
Isn't `make_timestamp` ANSI-complaint when the SQL config `spark.sql.ansi.enabled` is `true`?

Reply:
Yeah, I would remove this comment for ANSI-complient, try functions should just say that they return null on failure, and not actual result.